diff --git a/node_modules/undici-types/fetch.d.ts b/node_modules/undici-types/fetch.d.ts
index 7e94375..b2370f8 100644
--- a/node_modules/undici-types/fetch.d.ts
+++ b/node_modules/undici-types/fetch.d.ts
@@ -51,7 +51,7 @@ export class BodyMixin {
    * ```
    */
   readonly formData: () => Promise<FormData>
-  readonly json: () => Promise<unknown>
+  readonly json: () => Promise<any>
   readonly text: () => Promise<string>
 }
 
